README.md

```diff
@@ -1,73 +1,137 @@
-# Tarantool
+<p align="left">
+  <a href="https://tarantool.io/en/">
+    <img
+      alt="Tarantool"
+      src="https://i.imgur.com/BUYqImx.png"
+      width="400"
+    />
+  </a>
+</p>
 
-[![Build Status][travis-badge]][travis-url]
 [![Build Status][gitlab-ci-badge]][gitlab-ci-url]
 [![Code Coverage][coverage-badge]][coverage-url]
 [![Telegram][telegram-badge]][telegram-url]
-[![Slack][slack-badge]][slack-url]
-[![Gitter][gitter-badge]][gitter-url]
 [![Google Groups][groups-badge]][groups-url]
+[![Stackoverflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
+**A feature-rich in-memory database.** Our features set:
 
-Patch submissions and discussion of particular patches https://lists.tarantool.org/mailman/listinfo/tarantool-patches/
+* ACID
+* primary and secondary indexes
+* in-memory and disk engine
+* stored procedures
+* SQL
+* replication
+* sharding (as an external module)
+* and a wealth of modules and connectors from popular languages
 
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
+Another Tarantool feature is the ability to write code in Lua, a general purpose
+programming language, right inside the database. You can write custom expiration
+strategies and any business logic, and synchronize data with other data sources.
 
-Tarantool is an in-memory database and application server.
+You can use Tarantool to replace Redis when
+you need secondary indexes, a different structure of storage data, SQL, etc.
 
-Key features of the application server:
- * 100% compatible drop-in replacement for Lua 5.1,
-   based on LuaJIT 2.1.
-   Simply use `#!/usr/bin/tarantool` instead of
-   `#!/usr/bin/lua` in your script.
- * full support for Lua modules and a rich set of
-   own modules, including cooperative multitasking,
-   non-blocking I/O, access to external databases, etc
+For more details, please see the
+[Tarantool documentation](https://www.tarantool.io/en/doc/2.2/).
```

Comment:
Use `latest`, it is [implemented](https://github.com/tarantool/doc/issues/453) now.